pages/wi/2021/jobs.js

```diff
@@ -142,17 +151,17 @@ const jobs = () => {
   return (
     <>
       <NextSeo
-        title="WI 2020 Jobs - THAT Conference"
+        title="WI 2021 Jobs - THAT Conference"
```

Comment:
I see we have the event data, can this be made dynamic?

Reply:
@brettski they are in pages/wi/2021/jobs....so dynamic doesn't really make sense.